selfdrive/controls/controlsd.py

```diff
@@ -379,6 +379,8 @@ def update_events(self, CS):
         self.events.add(EventName.locationdTemporaryError)
       if not self.sm['liveParameters'].valid and not TESTING_CLOSET and (not SIMULATION or REPLAY):
         self.events.add(EventName.paramsdTemporaryError)
+      if not self.sm['liveParameters'].angleOffsetValid:
+        self.events.add(EventName.paramsdOffsetError)
```

Comment:
this needs an if/else or the alert priorities need to be adjusted

Reply:
Yes, I had a question here. Do you know if the TESTING_CLOSET and SIMULATION checks are still needed? Unclear why it is only on this one paramsd check, and not on all locationd checks too.